doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -714,29 +714,199 @@ general for HDDs, the bluestore throttle values are expected to be higher when
+The examples use :confval:`osd_mclock_max_capacity_iops_hdd`.
+
+Scenario 1: Config global set before deploying OSDs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When a global value is set before deploying the OSDs, the benchmark test during
+OSD deployment is skipped thus ensuring that the globally set value is in effect.
+
+The config global may be set before deploying OSDs but after enabling the Ceph
+monitor(s) using:
+
+.. prompt:: bash #
+
+  ceph config set global osd_mclock_max_capacity_iops_hdd 370
+
+Alternatively, the IOPS capacity may be specified under the *[global]* section of
+the Ceph configuration file during cluster deployment.
+
+A config dump after the OSD is deployed shows the global value in effect
+shown below:
+
+.. prompt:: bash #
+
+  ceph config dump
+
+::
+
+  WHO     MASK  LEVEL  OPTION                            VALUE       RO
+  global        basic  osd_mclock_max_capacity_iops_hdd  370.000000    
+
+.. prompt:: bash #
+
+  ceph config show osd.0 | grep osd_mclock
+
+::
+
+  osd_mclock_max_capacity_iops_hdd                 370.000000                               mon
+  ...
+
+Scenario 2: Config global set after deploying OSDs with individual values
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section shows the steps to globally override the IOPS capacity with OSDs
+already having individually scoped values in the mon store. Note that the
+individually scoped values are stored by the OSD itself after running the OSD
+bench test on start-up.
+
+For e.g., consider a deployment with 4 OSDs and the mon store having individual
+values like shown below:
+
+.. prompt:: bash #
+
+  ceph config dump
+
+::
+
+  WHO     MASK  LEVEL  OPTION                            VALUE       RO
+  osd.0         basic  osd_mclock_max_capacity_iops_hdd  379.197568    
+  osd.1         basic  osd_mclock_max_capacity_iops_hdd  400.903575    
+  osd.2         basic  osd_mclock_max_capacity_iops_hdd  398.303428    
+  osd.3         basic  osd_mclock_max_capacity_iops_hdd  419.035854    
+
+The config global is set using:
+
+.. prompt:: bash #
+
+  ceph config set global osd_mclock_max_capacity_iops_hdd 370
+
+The config dump now looks like:
+
+.. prompt:: bash #
+
+  ceph config dump
+
+::
+
+  WHO     MASK  LEVEL  OPTION                            VALUE       RO
+  global        basic  osd_mclock_max_capacity_iops_hdd  370.000000    
+  osd.0         basic  osd_mclock_max_capacity_iops_hdd  379.197568    
+  osd.1         basic  osd_mclock_max_capacity_iops_hdd  400.903575    
+  osd.2         basic  osd_mclock_max_capacity_iops_hdd  398.303428    
+  osd.3         basic  osd_mclock_max_capacity_iops_hdd  419.035854    
+
+But individual values for the OSDs are still not overridden due to the way
+the config subsystem is designed. See :ref:`ceph-conf-settings` for more details.
+For e.g., osd.0 still shows its individual value in-effect shown below:
+
```

Comment:
It is important to note that even though the global value is set after the per-OSD values, it does not override them.  Values configured for individual OSDs override the global value.  See :ref:`ceph-conf-settings` for more details.  For example, note that ``osd.0`` uses its individual value:

Reply:
Condensed the content to a single section and so this content is removed now. Please re-verify.